frontend/src/components/search/SearchSortBy.tsx

```diff
@@ -7,16 +7,16 @@ type SortOption = {
 };
 
 const SORT_OPTIONS: SortOption[] = [
-  { label: "Opportunity Number (Ascending)", value: "opportunityNumberAsc" },
+  { label: "Posted Date (Descending)", value: "postedDateDesc" },
+  { label: "Posted Date (Ascending)", value: "postedDateAsc" },
   { label: "Opportunity Number (Descending)", value: "opportunityNumberDesc" },
-  { label: "Opportunity Title (Ascending)", value: "opportunityTitleAsc" },
+  { label: "Opportunity Number (Ascending)", value: "opportunityNumberAsc" },
   { label: "Opportunity Title (Descending)", value: "opportunityTitleDesc" },
-  { label: "Agency (Ascending)", value: "agencyAsc" },
+  { label: "Opportunity Title (Ascending)", value: "opportunityTitleAsc" },
   { label: "Agency (Descending)", value: "agencyDesc" },
-  { label: "Posted Date (Ascending)", value: "postedDateAsc" },
-  { label: "Posted Date (Descending)", value: "postedDateDesc" },
-  { label: "Close Date (Ascending)", value: "closeDateAsc" },
+  { label: "Agency (Ascending)", value: "agencyAsc" },
```

Comment:
@andycochran fyi the new order

Reply:
all descending variants come before ascending now

Reply:
🤔 IDK if Z–A should come before A–Z. The date ones should be descending first. But the others makes more sense as ascending first. Can we also change them to: 

"Posted Date (newest)" `// descending`
"Posted Date (oldest)" 
Close Date (upcoming) `// descending`
Close Date (oldest)
Opportunity Title (A to Z) `// ascending`
Opportunity Title (Z to A)
Agency (A to Z)`// ascending`
Agency (Z to A)
Opportunity Number (ascending)
Opportunity Number (descending)

@crystabelrangel, do you have any input on ^these labels?

Reply:
What about for posted date something like `Posted Date (newest/descending)`

Reply:
However, the results set might not necessarily have any results with close dates in the future. Maybe just newest/oldest is fine, and we ignore the fact that "newest" may be in the future.

Reply:
Let's use: 

Posted Date (newest)
Posted Date (oldest)
Close Date (newest)
Close Date (oldest)
Opportunity Title (A to Z)
Opportunity Title (Z to A)
Agency (A to Z)
Agency (Z to A)
Opportunity Number (ascending)
Opportunity Number (descending)

^ in that order

Reply:
thanks folks - will try to get to this soon!

Reply:
![image](https://github.com/HHS/simpler-grants-gov/assets/93001277/840473b2-502e-4972-a109-02f7e86aa305)